templates/components/password-reset-success.jsx

```diff
@@ -0,0 +1,33 @@
+var React = require('react');
+var IconText = require('./icontext.jsx');
+
+var PasswordResetSuccess = React.createClass({
+  render: function() {
+    var message = (
+      <p>Sign in with your password below</p>
+    );
+
+    if (this.props.android) {
+      message = (
+        <p>
+          <span>
```

Comment:
I did this because React wraps plain text in a span, and the anchor tag following it doesn't flow properly